launch/launch/actions/execute_local.py

```diff
@@ -89,7 +91,7 @@ def __init__(
             'sigkill_timeout', default=5),
         emulate_tty: bool = False,
         output: SomeSubstitutionsType = 'log',
-        output_format: Text = '[{this.process_description.final_name}] {line}',
+        output_format: Text = DEFAULT_OUTPUT_FORMAT,
```

Comment:
It might be better to default to `None` here and use that as the logic to decide whether or not something other than default was passed. Thoughts?

Reply:
Due to changing the override behavior, this was resolved.